### DISCUSSION TITLE
WSL setup Issue

### DISCUSSION THREAD

**Discussion by Shiv2116** [Cyfrin/foundry-full-course-cu#55] [Q&A] [answered]
i have successfully install wsl in my windows but when opening the terminal in vscode I am facing  this issue
```
The terminal process failed to launch: Starting directory (cwd) "/root/solidity-course/C:\Users\hp" does not exist.
```

**Comment by cromewar** [comment, marked as answer]
Hello @Shiv2116 
the reason is because the communication between the folder paths form WSL and Windows is tricky, and also slower.
I just Highly recommend you to keep your project folders on your WSL instance, if you are not familiarized with UNIX command lines use this as reference:

1. `mkdir` to create a new folder.
2. `cd` to navigate between folders.
3. `ls` to see the contents of a folder.

so for example you can open the WSL terminal, create a folder for your project `mkdir folderName`, then navigate to it `cd folderName`, and finally open vscode inside of it using `code .`.
